tfx/proto/orchestration/executable_spec.proto

```diff
@@ -30,7 +30,7 @@ message BeamExecutableSpec {
   // Nested python executor spec.
   PythonClassExecutableSpec python_executor_spec = 1;
   // Args for Apache Beam pipeline.
-  repeated string beam_pipeline_args = 2;
+  repeated PlaceholderExpression beam_pipeline_args = 2;
```

Comment:
The best practice for protobuf is not changing the field type, because it is not backward compatible. We need to add a new field (maybe call it "beam_pipeline_args_ph" or "beam_pipeline_args_placeholder") and keep the original field (with a comment says it's deprecated). 

In other files, we need to change to use the new proto field.

Reply:
@kennethyang404 Made a change, please let me know if this was what you had in mind.